extraTs/.gitignore

```diff
@@ -0,0 +1 @@
+dist
```

Comment:
.gitIgnore file is created in root directory. Kindly remove it from here. Add dist to gitIgnore